js/tests/integration/bundle-modularity.js

```diff
@@ -1,7 +1,7 @@
-import 'popper.js'
 import Tooltip from '../../dist/tooltip'
+import '../../dist/carousel'
 
 window.addEventListener('load', () => {
-  [...document.querySelectorAll('[data-toggle="tooltip"]')]
+  [].slice.call(document.querySelectorAll('[data-toggle="tooltip"]'))
```

Comment:
Why change this `[...querySelectorAll()]` to `[].slice.call(querySelectorAll())` ? It's less readable.

Other solution: `Array.from(querySelectorAll())`

Reply:
It's mostly for consistency at this point since we still use this pattern across the docs.

Reply:
> consistency

Well, documentation uses `var` while internal JS code uses `const` :-)

What about keeping internal code "modern" (e.g `const`, `[...querySelectorAll()]`) and update documentation instead?

```JavaScript
// list-group.md
var triggerTabList = [].slice.call(document.querySelectorAll('#myTab a'))
triggerTabList.forEach(function (triggerEl) {
  var tabTrigger = new bootstrap.Tab(triggerEl)

  triggerEl.addEventListener('click', function (e) {
    e.preventDefault()
    tabTrigger.show()
  })
})

// popovers.md
var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-toggle="popover"]'))
var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
  return new bootstrap.Popover(popoverTriggerEl)
})
```

VS

```JavaScript
// list-group.md
document.querySelectorAll('#myTab a').forEach(triggerEl => {
  const tabTrigger = new bootstrap.Tab(triggerEl)

  triggerEl.addEventListener('click', e => {
    e.preventDefault()
    tabTrigger.show()
  })
})

// popovers.md
[...document.querySelectorAll('[data-toggle="popover"]')].map(triggerEl =>
  new bootstrap.Popover(triggerEl)
)
```

This syntax is supported by Bootstrap v5 minimum requirements: https://github.com/twbs/bootstrap/blob/v5.0.0-alpha2/.browserslistrc

Reply:
Like I said this is for consistency. The docs still use ES5 mostly. At some point we need to revisit this, but this is not the right time.

Reply:
Also, it needs to be done for the whole docs not just parts. So, it's not a trivial change especially since we currently don't lint the markdown/html files.

You could jump on helping, though, but I'd keep this issue as low priority because we have a lot more important things we need to address first :)